pages.es/osx/rubocop.md

```diff
@@ -1,6 +1,7 @@
 # rubocop
 
 > Analiza archivos de Ruby.
+> Más información: <https://docs.rubocop.org/rubocop/usage/basic_usage.html>.
```

Comment:
please move this page to the `common` folder. the same for the `pages.zh` version.